### PR TITLE
Command Line Arg to Disable Estop for Testing

### PR DESCRIPTION
**Pull request by sauravbanna**
### Description

Added a Command Line argument to disable all estop checks for testing

### Testing Done

Thunderscope builds and runs normally

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/thunderscope/robot_communication.py [hunk 1]
```diff
@@ -37,18 +38,20 @@ def __init__(
...
         :param interface: The interface to use
+        :param disable_estop: whether to disable estop checks (ONLY FOR TESTING LOCALLY)
         :param estop_path: The path to the estop
         :param estop_baudrate: The baudrate of the estop
 
         """
         self.sequence_number = 0
         self.last_time = time.time()
         self.current_proto_unix_io = current_proto_unix_io
+        self.current_mode = current_mode
```
Comment (LiCody) [inline src/software/thunderscope/robot_communication.py:49]: is this addition intentional?

**Review by LiCody — COMMENTED** [review]
one comment otherwise LGTM

File: src/software/thunderscope/robot_communication.py (hunk 1 above)
Comment (nimazareian) [reply src/software/thunderscope/robot_communication.py:49]: This looks to be an existing variable which was moved from lower down.

**Review by nimazareian — APPROVED** [review]
LGTM!

**Review by LiCody — APPROVED** [review]
LGTM
